<?xml version="1.0"?>

<launch>
  <include file="/opt/ros/indigo/share/gazebo_ros/launch/empty_world.launch">
   <!-- arg name="world" value="$(find hector_gazebo_worlds)/worlds/rolling_landscape_120m.world"/ --> 
  </include>
  
  <group ns="robot1">
	  <remap from="/robot1/cmd_vel" to="/cmd_vel"/>
    <param name="tf_prefix" value="robot1_tf" />
    <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
	  <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/>
	  <arg name="x" value="1" />
      <arg name="y" value="1" />
      <arg name="z" value="0" />
	  <arg name="name"  value="Robot1" />
    </include>
  </group>
  
  <group ns="robot2">
	  <remap from="/robot2/cmd_vel" to="/cmd_vel"/>
    <param name="tf_prefix" value="robot2_tf" />
    <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
      <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/>
      <arg name="x" value="2" />
      <arg name="y" value="2" />
      <arg name="z" value="0" />
      <arg name="name"  value="Robot2" />
    </include>
  </group>
  
  <group ns="robot3">
	  <remap from="/robot3/cmd_vel" to="/cmd_vel"/>
    <param name="tf_prefix" value="robot3_tf" />
    <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
      <arg name="model" value="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro"/>
      <arg name="x" value="3" />
      <arg name="y" value="3" />
      <arg name="z" value="0" />
      <arg name="name"  value="Robot3" />
    </include>
  </group>
  
    
 


</launch>
